src/commands.ts

```diff
@@ -8,6 +8,7 @@ import * as pify from 'pify';
 import * as vscode from 'vscode';
 import Config from './config';
 import Utils from './utils';
+import { pathToFileURL } from 'url';
```

Comment:
This import seems unused, can you remove it?

Reply:
Good catch, removed.